webrender/src/profiler.rs

```diff
@@ -947,126 +960,162 @@ impl Profiler {
         }
         renderer_timers.gpu_time.set(gpu_time);
 
-        self.draw_counters(&[&renderer_profile.frame_time], debug_renderer, true);
-        self.draw_counters(&[&renderer_profile.frame_counter], debug_renderer, true);
+        self.backend_time
+            .push(backend_profile.total_time.nanoseconds);
+        self.compositor_time
+            .push(renderer_timers.cpu_time.nanoseconds);
+        self.ipc_time
+            .push(backend_profile.ipc.total_time.nanoseconds);
+        self.gpu_time.push(gpu_time);
+        self.gpu_frames.push(gpu_time, gpu_timers);
 
-        self.draw_gpu_cache_bars(
-            &backend_profile.resources.gpu_cache,
-            debug_renderer,
-        );
+        if compact {
```

Comment:
those branches make we want a separate function for the long/short statistics

Reply:
Done